### DISCUSSION TITLE
[Feature] More from this time period

### DISCUSSION THREAD

**Discussion by Explorographer** [immich-app/immich#8685] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When using search, if you click on a photo, include a button: More from this timeline and when clicked it takes you to the searched photo on the timeline so you can see more photos from that time period. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4115
